Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -21,6 +21,7 @@ private enum ConfigurationKey: String {
     case UseNestedConfigs = "use_nested_configs" // deprecated
     case WhitelistRules = "whitelist_rules"
     case WarningThreshold = "warning_threshold"
+    case RequiredVersion = "required_version"
```

Comment:
"required_version" makes it sound like the versions need to match exactly. The check later on is a `>`. This should probably be renamed "minimum_version", "swiftlint_minimum_version", or "swiftlint_minimum" with the current implementation. If you add support for the `~>` operator, then maybe just "swiftlint_version". Thoughts @realm/swiftlint ?